javascript/javascript1/week1/js1homework.js

```diff
@@ -0,0 +1,119 @@
+//Excercise1
+const yearOfBirth = 1980;
+const futureYear = 2025;
+const userName = "Reenu";
+const currentAge = new Date().getFullYear() - yearOfBirth;
+const futureAge = futureYear - yearOfBirth;
+
+console.log(userName + " is " + currentAge + " years old now.");
+console.log("She will be " + futureAge + " years old in " + futureYear);
+
+//Excercise2
+//"1 dog year = 7 human years"
+
+const dogYearOfBirth = 2015;
+const dogYearFuture = 2025;
+let shouldShowResultInDogYears = true;
+const dogAgeInFuture = dogYearFuture - dogYearOfBirth;
+if (shouldShowResultInDogYears) {
+  console.log(
+    `Your dog will be ${dogAgeInFuture * 7} dog years old in ${dogYearFuture}`
+  );
+} else {
+  console.log(
+    `Your dog will be ${dogAgeInFuture} human years old in ${dogYearFuture}`
+  );
+}
+
+//Excercise 3
+//housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+const houseDimensionPeter = [8, 10, 10];
```

Comment:
Imagine you can put all values in the same array for both Peter and Julia, for example, 
`const volumeInMeters = [8*10*10,5*11*8];` `const gardenSizeInM2 = [100,70];`
where you know that volumeInMeters[0] is Peter's house volume, and another one is Julia's etc.
Can you refactor the rest of your code so you will have fewer lines of code?